Streetcode/Streetcode.WebApi/Utils/WebParsingUtils.cs

```diff
@@ -47,51 +42,51 @@ public async Task DownloadAndExtractAsync(string fileUrl, string zipPath, string
             clientHandler.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
             clientHandler.ServerCertificateCustomValidationCallback = (message, cert, chain, errors) => true;
 
-            using (var client = new HttpClient(clientHandler, false) { DefaultRequestHeaders = { }, Timeout = TimeSpan.FromSeconds(60) })
+            using var client = new HttpClient(clientHandler, false) { DefaultRequestHeaders = { }, Timeout = TimeSpan.FromSeconds(60) };
```

Comment:
Please write every property on a separate line, so that the maximum of 120 symbols per line is not broken.

Reply:
done